cfme/services/catalogs/catalog.py

```diff
@@ -1,115 +1,104 @@
-# -*- coding: utf-8 -*-
-from functools import partial
-
+from widgetastic.widget import Text
+from widgetastic_patternfly import Button, Input
 from navmazing import NavigateToAttribute, NavigateToSibling
-
-from cfme import web_ui
-from cfme.exceptions import DestinationNotFound
-from cfme.fixtures import pytest_selenium as sel
-from cfme.web_ui import accordion, flash, form_buttons, Input
-from cfme.web_ui import toolbar as tb, paginator, CheckboxTable, match_location
-from utils.appliance import Navigatable
-from utils.appliance.implementations.ui import navigator, navigate_to, CFMENavigateStep
-from utils.blockers import BZ
-from utils.pretty import Pretty
 from utils.update import Updateable
+from utils.pretty import Pretty
+from utils.appliance import Navigatable
+from utils.appliance.implementations.ui import navigator, CFMENavigateStep, navigate_to
 from utils import version
 
 
-cfg_btn = partial(tb.select, "Configuration")
-catalog_tree = partial(accordion.tree, "Catalogs")
+from . import ServicesCatalogView
 
-listview_table = CheckboxTable(table_locator='//div[@id="list_grid"]/table')
 
-item_multiselect = web_ui.MultiSelect(
-    "//select[@id='available_fields']",
-    "//select[@id='selected_fields']",
-    "//div[@id='column_lists']//a[contains(@href, 'button=right')]/img",
-    "//div[@id='column_lists']//a[contains(@href, 'button=left')]/img")
+class CatalogForm(ServicesCatalogView):
+    title = Text('#explorer_title_text')
 
-form = web_ui.Form(
-    fields=[('name_text', Input("name")),
-            ('description_text', Input("description")),
-            ('button_multiselect', item_multiselect),
-            ('add_button', form_buttons.add),
-            ('save_button', form_buttons.save),
-            ('cancel_button', form_buttons.cancel)])
+    name_text = Input(name='name')
+    description_text = Input(name="description")
 
-item_form = web_ui.Form(
-    fields=[('type_select', "//select[@id='st_prov_type']"),
-            ('name_text', Input("name")),
-            ('description_text', Input("description")),
-            ('display_checkbox', Input("display")),
-            ('add_button', form_buttons.add)])
+    add_button = Button('Add')
+    save_button = Button('Save')
+    cancel_button = Button('Cancel')
```

Comment:
**REQUIRED**: The CatalogForm child classes also specify the buttons. Currently it is customary to have them in the child classes only (for the different between add/save). So you should remove them here.